### PR TITLE
API Assignment -4

### PR DESCRIPTION
**Pull request by anushreeSonak**

### CODE REVIEW

File: APIAssignment/src/test/java/APIAssignments/PutBrand.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+    public PutBrand() {
+        try {
+            url = ConfigReader.getUrl();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private static Logger logger = Logger.getLogger("PutBrandList.class");
```
Comment (prashantkawa) [inline APIAssignment/src/test/java/APIAssignments/PutBrand.java]: Use final here

File: APIAssignment/src/test/java/APIAssignments/PutBrand.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    @Test(priority = 1)
+    public void getResponseCode() {
+        var response = given().when().put(url).then().extract().asString();
+        JsonPath jsonResponse = new JsonPath(response);
+        var responseCode = jsonResponse.get("responseCode");
+        logger.info("responseCode: " + responseCode);
+        Assert.assertEquals(responseCode, 405);
+    }
+
+
```
Comment (prashantkawa) [inline APIAssignment/src/test/java/APIAssignments/PutBrand.java]: remove extra spaces

File: APIAssignment/src/test/java/APIAssignments/PutBrand.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    @Test(priority = 2)
+    public void getResponseMessage() {
+        var response = given().when().put(url).then().extract().asString();
+        JsonPath jsonResponse = new JsonPath(response);
+        var responseMessage = jsonResponse.get("message");
+        logger.info("Response Message is: " + responseMessage);
+        Assert.assertEquals(responseMessage, "This request method is not supported.");
+    }
+}
+
```
Comment (prashantkawa) [inline APIAssignment/src/test/java/APIAssignments/PutBrand.java]: same here

File: APIAssignment/src/test/java/APIAssignments/PutBrand.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
+package APIAssignments;
+
+import io.restassured.path.json.JsonPath;
+import org.apache.log4j.PropertyConfigurator;
+import org.testng.Assert;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
```
Comment (prashantkawa) [inline APIAssignment/src/test/java/APIAssignments/PutBrand.java:9]: Add readme file for problem statement and output

**Review by prashantkawa — CHANGES_REQUESTED** [review]

File: APIAssignment/src/test/java/APIAssignments/PutBrand.java [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+import java.util.logging.Logger;
```
Comment (prashantkawa) [inline APIAssignment/src/test/java/APIAssignments/PutBrand.java]: use Logger from apache.log4j library

**Review by prashantkawa — CHANGES_REQUESTED** [review]

**Review by prashantkawa — APPROVED** [review]
Approving PR

**Review by vrushalee123 — APPROVED** [review]

**Pull request merged**